iconfont/PXBlueIcons.css

```diff
@@ -1,10 +1,10 @@
 @font-face {
     font-family: "PXBlueIcons";
-    src: url("./PXBlueIcons.eot?46d0c2acad099763c41c7e3494941c24?#iefix") format("embedded-opentype"),
-url("./PXBlueIcons.woff2?46d0c2acad099763c41c7e3494941c24") format("woff2"),
-url("./PXBlueIcons.woff?46d0c2acad099763c41c7e3494941c24") format("woff"),
-url("./PXBlueIcons.ttf?46d0c2acad099763c41c7e3494941c24") format("truetype"),
-url("./PXBlueIcons.svg?46d0c2acad099763c41c7e3494941c24#PXBlueIcons") format("svg");
+    src: url("./PXBlueIcons.eot?89ca8fa3c6f8dc0fe1fcb6ff7dead92f?#iefix") format("embedded-opentype"),
```

Comment:
Why is there an updated version of the icon font?

Reply:
I don’t know; I ran yarn build and it updates automatically for me

Reply:
I did `yarn build:all again`, and the files are still mysteriously updating themselves

Reply:
I assumed that the hash wouldn't change unless there are icon changes, but it's possible that it's just a random string. Not a big deal